operations/make-private.js

```diff
@@ -0,0 +1,6 @@
+var setAcl = require('../set-acl.js');
+
+module.exports = setAcl(function getAcl(bucket, key) {
```

Comment:
kind of pedantic, but I think `setAcl` should be named something like `setAclFactory` since it just returns a function